app/presenters/world_index_graphql_presenter.rb

```diff
@@ -0,0 +1,15 @@
+class WorldIndexGraphqlPresenter < WorldIndexPresenter
```

Comment:
Is this kind of inheritance okay long term? I'm just wondering what happens if we want to make graphql the default?

Reply:
My theory was to reduce the deviation from what's already there as much as possible, so both behave the same.

When we want to make GraphQL the default, we can just move the methods from this class into `WorldIndexPresenter`.